homeassistant/helpers/config_validation.py

```diff
@@ -219,6 +221,17 @@ def validate(obj):
     return validate
 
 
+def validate_values(valid_values):
```

Comment:
SENSOR_TYPES.keys() is already a list, and is fine.

It is the config input that should should be converted from a set to a list.
Because this is a valid configuration:
  monitored_conditions:
    - temperature
    - symbol
    - precipitation
    - windSpeed
    - pressure
    - windDirection

And the list of items seams to be a type